### ISSUE TITLE
description of row ids not clear

### ISSUE THREAD

**Issue by nickreich** [mlr-org/mlr3book#127]
In the Tasks section (2.2) there is a section that is unclear about how rows are indexed. 
It says:
```
In `r mlr_pkg("mlr3")`, each row (observation) has a unique identifier, stored as an `intger()`.
These can be passed as arguments to the `$data()` method to select specific rows:
```
However, it's not clear to me what an `intger()` is? maybe this is supposed to be an integer, but then in the next example, the task row_ids are characters. Maybe rephrase to something like this?
```
In `r mlr_pkg("mlr3")`, each row (observation) has a unique identifier. Sometimes these unique identifiers are characters and sometimes integers.
```

**Comment by mllg** [comment]
Character row ids will not be supported by the next release, I've cleaned up the mentioned part.
